service-deployment/clusterconfig-example.yaml

```diff
@@ -63,6 +63,19 @@ clusterinfo:
     configmapname: hadoop-configuration
     hadoop_vip: 10.0.3.9
 
+  # Step 1 of 4 to set up Hadoop queues.
+  # Define all virtual clusters, equivalent concept of Hadoop queues.
+  virtualClusters:
+    default:
+      description: The default virtual cluster for testing purpose only.
+      weight: 20
```

Comment:
>weight [](start = 6, length = 6)

Maybe call it GPUs is simpler to understand for cluster admins?

Reply:
Indeed. But it is better to have an additional mechanism to calculate the percentage automatically according to the # of gpus defined here and the total # of gpus defined in clusterconfig.yaml file. I'll figure out a solution.